lua/caramba/orchestrator.lua

```
@@ -0,0 +1,278 @@
+	end)
+end
+
+--- Post-process an assistant response: store memory and update plan
+--- @param user_message string
+--- @param assistant_text string
+function M.postprocess_response(user_message, assistant_text)
+	if assistant_text and assistant_text ~= '' then
+		local ctx = context.collect({})
+		local tags = { "caramba", "assistant_response" }
+		if ctx and ctx.language then table.insert(tags, ctx.language) end
+		memory.store(assistant_text:sub(1, 2000), { 
```

Luacheck (1.2.0)

[warning] line contains trailing whitespace

(W612)